### ISSUE TITLE
Blogspot.com blogs breaks video stream

### ISSUE THREAD

**Issue by terrorist96** [EFForg/https-everywhere#8113]
http://streamfare.com/fox_news_2.html
http://streamfare.com/univisioni.html
http://streamfare.com/2abc.html
etc

`Blogspot.com blogs` breaks video.

**Comment by Bisaloo** [comment]
@terrorist96, this seems fixed for test url 1 and I can reproduce without HTTPS Everywhere for test urls 2 & 3. Can you check?

**Comment by terrorist96** [comment]
Confirmed. But I found another broken one for you where disabling `Blogspot.com blogs` fixes it:
http://streamfare.com/telemundo.html

**Comment by Bisaloo** [comment]
I can reproduce with HTTPS Everywhere disabled as well. Can you add your console output to this issue please?

**Comment by terrorist96** [comment]
Sure. Keeping the console open, then refreshing the page, shows three different instances of errors, but they disappear.
When I first load the page I see:
```
telemundo.html:304 GET https://www.googletagmanager.com/gtag/js?id=UA-106776484-3 net::ERR_BLOCKED_BY_CLIENT
telemundo.html:370 GET http://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js net::ERR_BLOCKED_BY_CLIENT
telemundo.html:386 GET http://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js net::ERR_BLOCKED_BY_CLIENT
cb=gapi.loaded_0:164 Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://accounts.google.com') does not match the recipient window's origin ('http://streamfare.com').
(anonymous) @ cb=gapi.loaded_0:164
setTimeout (async)
call @ cb=gapi.loaded_0:164
call @ cb=gapi.loaded_0:176
(anonymous) @ cb=gapi.loaded_1:288
_.Xw @ cb=gapi.loaded_1:308
dx @ cb=gapi.loaded_1:288
callback @ platform.js:61
(anonymous) @ platform.js:18
Oa @ platform.js:10
b @ platform.js:18
E.(anonymous function).E.(anonymous function) @ platform.js:18
c @ platform.js:18
(anonymous) @ platform.js:18
Oa @ platform.js:10
b @ platform.js:18
E.(anonymous function) @ platform.js:18
B.(anonymous function) @ platform.js:18
(anonymous) @ cb=gapi.loaded_0:1
```
Then the above error gets deleted and I then see:
```
telemundo-222.html:750 GET https://pagead2.googlesyndication.com/pagead/js/google_top_exp.js net::ERR_BLOCKED_BY_CLIENT
(anonymous) @ telemundo-222.html:750
(anonymous) @ telemundo-222.html:751
telemundo-222.html:1 Mixed Content: The page at 'https://2livewatch.blogspot.com/2016/10/telemundo-222.html' was loaded over HTTPS, but requested an insecure plugin resource 'http://static-cdn1.ustream.tv/swf/live/viewerqos:23.swf'. This request has been blocked; the content must be served over HTTPS.
```
Then the above error gets deleted and is replaced by the following, which remains on screen and doesn't get deleted.
```
img.secureserver.net/t/1/tl/event?cts=1519260192070&tce=1519260189213&tcs=1519260189140&tdc=1519260191867&tdclee=1519260189667&tdcles=1519260189663&tdi=1519260189663&tdl=1519260189334&tdle=1519260189140&tdls=1519260189139&tfs=1519260189137&tns=1519260189133&trqs=1519260189213&tre=1519260189296&trps=1519260189289&tles=1519260191867&tlee=1519260191890&nt=1&ht=perf&dh=streamfare.com&ua=Mozilla%2F5.0%20(Windows%20NT%206.1%3B%20Win64%3B%20x64)%20AppleWebKit%2F537.36%20(KHTML%2C%20like%20Gecko)%20Chrome%2F64.0.3282.140%20Safari%2F537.36&vci=324843992&cv=1.0.6&z=392876766&vg=1ef347c8-3849-4ec0-b078-090fed38ec6e&vtg=1ef21970-3967-4eeb-9c45-0b42ed5215cf&ap=cpsh&trfd=%7B%22cts%22%3A1519260189662%2C%22tccl.baseHost%22%3A%22secureserver.net%22%2C%22ap%22%3A%22cpsh%22%2C%22server%22%3A%22p3plcpnl0533%22%7D&dp=%2Ftelemundo.html:1 GET http://img.secureserver.net/t/1/tl/event?cts=1519260192070&tce=1519260189213&tcs=1519260189140&tdc=1519260191867&tdclee=1519260189667&tdcles=1519260189663&tdi=1519260189663&tdl=1519260189334&tdle=1519260189140&tdls=1519260189139&tfs=1519260189137&tns=1519260189133&trqs=1519260189213&tre=1519260189296&trps=1519260189289&tles=1519260191867&tlee=1519260191890&nt=1&ht=perf&dh=streamfare.com&ua=Mozilla%2F5.0%20(Windows%20NT%206.1%3B%20Win64%3B%20x64)%20AppleWebKit%2F537.36%20(KHTML%2C%20like%20Gecko)%20Chrome%2F64.0.3282.140%20Safari%2F537.36&vci=324843992&cv=1.0.6&z=392876766&vg=1ef347c8-3849-4ec0-b078-090fed38ec6e&vtg=1ef21970-3967-4eeb-9c45-0b42ed5215cf&ap=cpsh&trfd=%7B%22cts%22%3A1519260189662%2C%22tccl.baseHost%22%3A%22secureserver.net%22%2C%22ap%22%3A%22cpsh%22%2C%22server%22%3A%22p3plcpnl0533%22%7D&dp=%2Ftelemundo.html net::ERR_BLOCKED_BY_CLIENT
Image (async)
f @ tcc_l.combined.1.0.6.min.js:57
e @ tcc_l.combined.1.0.6.min.js:52
b @ tcc_l.combined.1.0.6.min.js:42
push @ tcc_l.combined.1.0.6.min.js:62
c @ tcc_l.combined.1.0.6.min.js:28
b @ tcc_l.combined.1.0.6.min.js:26
setTimeout (async)
(anonymous) @ tcc_l.combined.1.0.6.min.js:36
```

**Comment by Bisaloo** [comment]
It looks like MCB. This should be fixed in #8113. You test this with the debugging tool (see https://github.com/EFForg/https-everywhere/issues/13731#issue-276704233).

**Comment by Bisaloo** [comment]
@terrorist96, do you have any other examples for this issue? We're unsure about the scope of the exclusion to add.
